packages/cactus-plugin-ledger-connector-ethereum/src/test/typescript/integration/geth-invoke-web3-method-v1.test.ts

```diff
@@ -167,43 +167,43 @@ describe("invokeRawWeb3EthMethod Tests", () => {
   });
 
   test("invokeRawWeb3EthMethod with missing arg throws error (getBlock)", async () => {
+
+    //did not refactor because the test is not failing. 
     try {
       const connectorResponse = connector.invokeRawWeb3EthMethod({
         methodName: "getBlock",
       });
 
       await connectorResponse;
+      //This test is actually passing, but the statement below is not being printed. 
       fail("Calling getBlock with missing argument should throw an error");
```

Comment:
@ashnashahgrover Remove the try-catch block and assert success in that case. Makes the test less confusing

Reply:
See:

https://github.com/hyperledger/cacti/issues/3487